dropwizard-hibernate/src/test/java/io/dropwizard/hibernate/PersistenceExceptionMapper.java

```diff
@@ -0,0 +1,37 @@
+package io.dropwizard.hibernate;
+
+import org.hibernate.exception.DataException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.persistence.PersistenceException;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import javax.ws.rs.ext.Providers;
+
+@Provider
+public class PersistenceExceptionMapper implements ExceptionMapper<PersistenceException> {
```

Comment:
Copy-paste the contents. Unfortunately, previous exception mappers become invalid without it, because Hibernate changed its exceptions hierarchy.